### PR TITLE
GM: enforce steering command message timing

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/car/body/carcontroller.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def deadband_filter(torque, deadband):
       torque -= deadband
     return torque
 
-  def update(self, CC, CS):
+  def update(self, CC, CS, now_nanos):
```
Comment (sshane) [inline selfdrive/car/body/carcontroller.py:38]: `log_mono_time`?

File: selfdrive/controls/controlsd.py [hunk 2]
```diff
@@ -731,7 +731,7 @@ def publish_logs(self, CS, start_time, CC, lac_log):
 
     if not self.read_only and self.initialized:
       # send car controls over can
-      self.last_actuators, can_sends = self.CI.apply(CC)
+      self.last_actuators, can_sends = self.CI.apply(CC, int(sec_since_boot() * 1e9))
```
Comment (sshane) [inline selfdrive/controls/controlsd.py:734]: we can also make a messaging helper function to get log mono time, copied what it does now

**Comment by sshane** [comment]
We can also get rid of `self.sent_lka_steering_cmd` if we check the CAN parser timestamp, I'll do that cleanup in another PR

**Review by adeebshihadeh — APPROVED** [review]

**Comment by sshane** [comment]
Working as expected. With and without at 50Hz:

Screenshot from 2023-02-09 15-20-02
Screenshot from 2023-02-09 15-21-57

**Pull request merged**